test-data/unit/check-columns.test

```diff
@@ -326,6 +336,12 @@ if int():
 main:2:11: error: Syntax error in type annotation
 main:2:11: note: Suggestion: Is there a spurious trailing comma?
 
+[case testColumnSyntaxErrorInTypeAnnotation2]
+if int():
+        x = 0  # type: x y
+[out]
+main:2:9: error: syntax error in type comment 'x y'
```

Comment:
It is not clear what is 9th column here? Maybe keep the intent as 4, otherwise there are two 8-column offsets, the indent itself and also the length of `# type: `.

Reply:
A good point, updated.